encoders/src/test/java/au/csiro/pathling/encoders/FhirEncodersTest.java

```diff
@@ -107,8 +105,9 @@ public static void setUp() {
         ENCODERS_L0.of(Observation.class));
     decodedObservation = observationsDataset.head();
 
+    // TODO: Uncomment with contained resources are supported
```

Comment:
I don't think we are ever going to support the encoding of resources that contain other resources.

We may support the unbundling of contained resources within a create, update, batch or transaction operation - but this would not involve the encoders. The resources would already be "uncontained" by the time that they hit the encoders.

Reply:
Ok. I have removed support for container resources from FhirEncoders and all references to it from v2 encoders.